src/index.js

```diff
@@ -14,6 +12,51 @@ function isRedo(e) {
   return (e.ctrlKey || e.metaKey) && e.keyCode === (e.shiftKey ? KEYCODE_Z : KEYCODE_Y)
 }
 
+function getSelection (el) {
+  var start, end, rangeEl, clone;
+
+  if( el.selectionStart !== undefined){
```

Comment:
nit: `if (typeof el.selectionStart !== 'undefined') {` (spacing should follow the rest of the codebase)